src/main/js/main_html5.js

```diff
@@ -1195,6 +1195,11 @@ require("../../../html5-common/js/utils/environment.js");
                       _video.currentTime);
           _.defer(raiseEndedEvent);
         }
+        if(OO.isSafari && !OO.isIos && !_video.ended && Math.round(_video.currentTime) === Math.round(_video.duration))
+        {
+          this.controller.notify(this.controller.EVENTS.SEEKED);
```

Comment:
why do we need to raise the seeked event?  Does it not work fine without it?  In the logs i had seen seeked event was already raised